app/components/markdown/error_boundry.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Text} from 'react-native';
+
+import {makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type State = {
+    hasError: boolean;
+}
+
+type Props = {
+    children: JSX.Element;
+    error: string;
+    theme: Theme;
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    error: {
+        color: theme.errorTextColor,
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+        fontStyle: 'italic',
+        ...typography('Body', 100),
+    },
+}));
+
+class ErrorBoundry extends React.PureComponent<Props, State, any> {
```

Comment:
```suggestion
class ErrorBoundary extends React.PureComponent<Props, State, any> {
```

Also on the file name.